packages/nds-tag/scss/tag.scss

```diff
@@ -46,17 +50,14 @@ $nds-tag-border-radius: em($nds-spacing-xx-small);
 
   &--alpha {
     background-color: $nds-colour-alpha;
-    color: $nds-colour-text-inverse;
   }
 
   &--beta {
     background-color: $nds-colour-beta;
-    color: $nds-colour-text-inverse;
   }
 
   &--live {
-    background-color: $nds-colour-live;
-    color: $nds-colour-text-inverse;
+    background-color: $nds-colour-success;
```

Comment:
The live tag should continue to use the `$nds-colour-live` variable. If there are colour contrast issues then let's update the colour var instead.